skycoin/skywire
# Fix/syslog

Did you run `make format && make check`?
yes

Fixes #983 

 Changes:	
- Added `syslog-ng.conf` file 
- Added main syslog logger to `NewProcLogger()`, `disc.NewHTTP()`, `dmsgc`,  and `ClientFactory`

How to test this PR:
1. Checkout [#dmsg-134](https://github.com/skycoin/dmsg/pull/134) in dmsg
2. In skywire uncomment `replace github.com/skycoin/dmsg => ../dmsg` in `go.mod` and run
    ```
    make dep
    ```
3. Run
    ```
    make build
    ```
    ```
    sudo syslog-ng -f ./init/syslog-ng.conf -edv
    ```
4. In a new terminal run
    ```
    ./skywire-visor -c skywire-config.json --syslog localhost:514
    ```
5. Check the logs in a new terminal via
    ```
    sudo cat /var/log/skywire
    ```

Needs:
[#dmsg-134](https://github.com/skycoin/dmsg/pull/134)
